sw/host/provisioning/ft_lib/src/lib.rs

```diff
@@ -165,49 +165,23 @@ pub enum KeyWrapper {
     CkmsKey(String),
 }
 
-fn extract_rma_unlock_token(
-    transport: &TransportWrapper,
-    host_ecc_sk: PathBuf,
-    timeout: Duration,
-) -> Result<()> {
+fn process_rma_unlock_token(transport: &TransportWrapper, timeout: Duration) -> Result<()> {
     let uart = transport.uart("console")?;
 
-    // Load host (HSM) generated ECC keys.
-    let host_sk = SecretKey::<NistP256>::read_pkcs8_der_file(host_ecc_sk)?;
-    let host_pk = PublicKey::<NistP256>::from_secret_scalar(&host_sk.to_nonzero_scalar());
-
-    // Format host ECC public key to inject it into the device.
-    // Note: we trim off the first byte of SEC1 formatted public key as these are not part
-    // of the key bytes, rather this byte just indicates if the key was compressed or not.
-    let host_pk_sec1_bytes = host_pk.to_sec1_bytes();
-    let num_coord_bytes: usize = (host_pk_sec1_bytes.len() - 1) / 2;
-    let mut host_pk_x = host_pk_sec1_bytes.as_ref()[1..num_coord_bytes + 1]
-        .chunks(4)
-        .map(|bytes| u32::from_be_bytes(bytes.try_into().unwrap()))
-        .collect::<ArrayVec<u32, 8>>();
-    let mut host_pk_y = host_pk_sec1_bytes.as_ref()[num_coord_bytes + 1..]
-        .chunks(4)
-        .map(|bytes| u32::from_be_bytes(bytes.try_into().unwrap()))
-        .collect::<ArrayVec<u32, 8>>();
-    host_pk_x.reverse();
-    host_pk_y.reverse();
-    let rma_token_wrapping_pubkey = EccP256PublicKey {
-        x: host_pk_x,
-        y: host_pk_y,
+    let mut token = ArrayVec::<u32, 4>::new();
+
+    token = (0..token.len())
+        .map(|_| rand::random::<u32>())
```

Comment:
Rather than generate random tokens, could we make the token an input to the FT test harness (similar to how the `host_sk` was before), and put it in `sw/device/silicon_creator/manuf/base/provisioning_inputs.bzl`? The reason is because the FT test harness (sw/host/provisioning/ft/src/main.rs) will be used by benchtop provisioning flows during initial chip bringup, and we want the HSM in the provisioning infrastructure to generate the LC tokens and then just pass them along to the test harness as inputs.